app/models/event.rb

```diff
@@ -16,6 +19,15 @@ class Event < ActiveRecord::Base
   # fileupload
   mount_uploader :image, PictureUploader
 
+  validate :expiration_date_cannot_be_in_the_past
+  validates :title, presence: true, length: { in: 5..250 }
+  validates :description, presence: true, length: { in: 20..1000 }
+  validates_presence_of :start_date, :end_date, :category_id
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.